Makefile

```diff
@@ -0,0 +1,49 @@
+artifact_name       := overseas-entities
```

Comment:
Probably should be `overseas-entities-web`?

Reply:
See: https://github.com/companieshouse/confirmation-statement-web/pull/6/files